runtime/block_quantization_kernels.cu

```diff
@@ -108,6 +115,10 @@ __device__ void block_quantize_to_nvfp4(
   // Convert back from FP8 to float using __e4m32float
   float clamped_max_converted = __e4m32float(clamped_max_fp8);
 
+  if constexpr (USE_GLOBAL_SCALE) {
+    clamped_max_converted = clamped_max_converted * global_scale[0];
```

Comment:
what's going on here? looks like we are reverting the ` / global_scale[0]`.

Reply:
Let me know if this better.
I tried to copy from:
https://github.com/nvidia/fuser/blob/pbasu_fp4_global_scaling/tests/python/direct_utils/narrow_precision.py#L125

Reply:
Oh you are right. Verified that with sglang and transformer engine implementation.

I forgot how nvfp4 quantization was done in the first place. :face_exhaling: 
But the update looks right to me.